test_util/aws.py

```diff
@@ -99,6 +97,18 @@ def create_stack(self, name, template_url, parameters, deploy_timeout=60):
             # so that we may pass parameters directly from yaml (which parses numbers as non-strings)
             Parameters=param_dict_to_aws_format(parameters))
 
+    def create_stack_from_string(self, name, template, parameters, deploy_timeout=60):
```

Comment:
By "pass straight through" I meant defer choosing/validating `TemplateBody` and `TemplateURL` to boto. See https://github.com/boto/boto/blob/abb38474ee5124bb571da0c42be67cd27c47094f/boto/cloudformation/connection.py#L229